app/src/main/java/phone/vishnu/quotes/fragment/FontSizeFragment.java

```diff
@@ -38,13 +39,19 @@ public void onCreate(@Nullable Bundle savedInstanceState) {
     @Override
     public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
         View inflate = inflater.inflate(R.layout.fragment_font_size, container, false);
-
+        defaultSize=inflate.findViewById(R.id.default_Size);
         sharedPreferenceHelper = new SharedPreferenceHelper(requireContext());
 
         sizeSlider = inflate.findViewById(R.id.fontSizeSlider);
         sizeSlider.setLabelFormatter(value -> (int) value + "sp");
         sizeSlider.setValue(sharedPreferenceHelper.getFontSizePreference());
 
+//        code to make size back to default(24sp)
+        defaultSize.setOnClickListener(v -> {
+            sharedPreferenceHelper.setFontSizePreference(24);
+            ((MainActivity) requireActivity()).updateViewPager();
+            sizeSlider.setValue(24);
+        });
```

Comment:
Move this click listener to `onViewCreated` (preferably below `sizeSlider`'s change listener)